src/components/Filterable-Form/FilterableForm.jsx

```diff
@@ -64,92 +68,92 @@ const FilterableForm = () => {
                 <label htmlFor="ap_filter-professionals">
-                    <input type="checkbox" id="ap_filter-professionals" value="Professionals" className="ap_input-field" />Professionals
+                    <input type="checkbox" id="ap_filter-professionals" name="filterByCategory" value="Professionals" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Professionals
                 </label>
                 <label htmlFor="ap_filter-saree">
-                    <input type="checkbox" id="ap_filter-saree" value="Saree" className="ap_input-field" />Saree
+                    <input type="checkbox" id="ap_filter-saree" name="filterByCategory" value="Saree" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Saree
                 </label>
                 <label htmlFor="ap_filter-shoes">
-                    <input type="checkbox" id="ap_filter-shoes" value="Shoes" className="ap_input-field" />Shoes
+                    <input type="checkbox" id="ap_filter-shoes" name="filterByCategory" value="Shoes" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Shoes
                 </label>
                 <label htmlFor="ap_filter-watches">
-                    <input type="checkbox" id="ap_filter-watches" value="Watches" className="ap_input-field" />Watches
+                    <input type="checkbox" id="ap_filter-watches" name="filterByCategory" value="Watches" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Watches
                 </label>
                 <label htmlFor="ap_filter-waterBottles">
-                    <input type="checkbox" id="ap_filter-waterBottles" value="Water Bottles" className="ap_input-field" />Water Bottles
+                    <input type="checkbox" id="ap_filter-waterBottles" name="filterByCategory" value="Water Bottles" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Water Bottles
                 </label>
                 <label htmlFor="ap_filter-western">
-                    <input type="checkbox" id="ap_filter-western" value="Western" className="ap_input-field" />Western
+                    <input type="checkbox" id="ap_filter-western" name="filterByCategory" value="Western" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Western
                 </label>
                 <label htmlFor="ap_filter-winter">
-                    <input type="checkbox" id="ap_filter-winter" value="Winter" className="ap_input-field" />Winter
+                    <input type="checkbox" id="ap_filter-winter" name="filterByCategory" value="Winter" className="ap_input-field" onClick={(e) => dispatch({ type: e.target.name, payload: e.target.value })} />Winter
                 </label>
```

Comment:
You can define an array then map through the array and return the label and input

Reply:
oh yaa, you have a point there, will try doing it with arrays